provider/pkg/provider/provider.go

```diff
@@ -1206,6 +1185,41 @@ func (k *kubeProvider) Check(ctx context.Context, req *pulumirpc.CheckRequest) (
 	return &pulumirpc.CheckResponse{Inputs: autonamedInputs, Failures: failures}, nil
 }
 
+// helmHookWarning logs a warning if a Chart contains unsupported hooks. The warning can be disabled by setting
+// the PULUMI_DISABLE_HELM_HOOK_WARNING environment variable.
```

Comment:
Should we make this a provider config field which can be set through the environment variable? We have existing support for this. Somewhat prefer following a single standard around manipulating provider behavior.

Reply:
Yeah, probably so. Will update.